### PR TITLE
Updated place group interaction

### PR DESCRIPTION
**Pull request by joewalsh**
This updates the places group interaction, but also has a number of other fixes. Merging this doesn't commit to the new interaction, it's simple to return to the old style.

### CODE REVIEW

**Comment by joewalsh** [comment]
jenkins test this please

**Comment by joewalsh** [comment]
jenkins test this please

File: Wikipedia/Code/ArticlePlaceGroupViewController.swift [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+        tintView.image = image
+        UIGraphicsEndImageContext()
+    }
+    
+    func show(center: CGPoint) {
+        self.center = center
+       
+        drawGradient(center: center)
+        
+        let rotationTransform = CGAffineTransform(rotationAngle: CGFloat(-0.99*M_PI))
```
Comment (amrox) [inline Wikipedia/Code/ArticlePlaceGroupViewController.swift]: Curious why the `-0.99` and `0.99` below

Comment (montehurd) [inline Wikipedia/Code/ArticlePlaceGroupViewController.swift]: Same.

Comment (joewalsh) [inline Wikipedia/Code/ArticlePlaceGroupViewController.swift]: To force it to rotate in a certain direction. Exactly M_PI and negative M_PI don't go the same way, this way it forces clockwise out and counter-clockwise back in.

Comment (montehurd) [inline Wikipedia/Code/ArticlePlaceGroupViewController.swift]: Ah, maybe something like `let forcedClockwiseAngle = 0.99` or some such to make that explicit?

**Pull request merged**